pkg/controller/managedresource/merger.go

```diff
@@ -15,6 +15,7 @@
 package managedresource
 
 import (
+	v1alpha1 "github.com/gardener/gardener-resource-manager/api/resources/v1alpha1"
```

Comment:
Usually, we use the pattern `<api-group><version>` as an import alias (see the other imports below), which makes things more readable.
```suggestion
	resourcesv1alpha1 "github.com/gardener/gardener-resource-manager/api/resources/v1alpha1"
```